scapy/contrib/eigrp.py

```diff
@@ -1,16 +1,7 @@
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This should be GPL-2.0-or-later

Reply:
You can have MIT with GPLv2. I'd say that's fine